server/src/controllers/buttonController.js

```diff
@@ -108,4 +105,124 @@ const saveJob = (req, res, data) => {
     ////////////////////////////////////////////////////////////////////
 }
 
+const locationButtons = (req, res, data) => {
+  let id = data.user.id;
+  let teamId = data.team.id;
+
+  User.find({
```

Comment:
Should we continue using rp with our API instead of calling methods directly on the Model that is not associated with the controller?